apps/portals/nf/src/config/synapseConfigs/tools.tsx

```diff
@@ -52,6 +53,11 @@ export const toolsCardConfiguration: CardConfiguration = {
   },
   secondaryLabelLimit: 4,
   genericCardSchema: toolsSchema,
+  cardTypeAdornment: (schema, data) => {
```

Comment:
Add a `cardTypeAdornment` on the tool card in NF.  If value is available, use a new NF specific `CompletenessIndicator` component